src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1028,6 +1030,7 @@ private void AggregateRenderPackages(IEnumerable<HelixRenderPackage> packages)
 
                         pointGeometry3D.Geometry = points;
                         pointGeometry3D.Name = baseId;
+                        pointGeometry3D.MouseDown3D += pointGeometry3D_MouseDown3D;
```

Comment:
Re-enabled geometry selection for `PointGeometryModel3D` type to be able to select only point geometry for now. This is so that a point selected in geometry view can highlight the corresponding Point node.